Gemfile

```diff
@@ -39,7 +47,7 @@ gem "proxifier", git: "https://github.com/chef/ruby-proxifier", branch: "lcg/rub
 # Everything except AIX and Windows
 group(:ruby_shadow) do
   # if ruby-shadow does a release that supports ruby-3.0 this can be removed
-  gem "ruby-shadow", git: "https://github.com/chef/ruby-shadow", branch: "lcg/ruby-3.0", platforms: :ruby
+  gem "ruby-shadow", git: "https://github.com/chef/ruby-shadow", branch: "lcg/ruby-3.0", platforms: :ruby unless RUBY_PLATFORM.match?(/x64-mingw-ucrt/i)
```

Comment:
We'll probably need to use an expression that catches both ruby 3.1 (x64-mingw-ucrt) and ruby 3.0 (x64-mingw32).

Reply:
Done